hidimstat/ada_svr.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+from hidimstat.stat_tools import pval_from_scale
+
+
+def ada_svr(X, y, rcond=1e-3):
```

Comment:
This code is copied from the existing codebase, I guess ?

Reply:
No, This was the code of the library. I just commented it.

Reply:
It's new because, rename the file.

Reply:
I just want to avoid code duplication.